src/snapshot/Cargo.toml

```diff
@@ -2,17 +2,18 @@
 name = "snapshot"
 version = "0.1.0"
 authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
+autobenches = false
```

Comment:
I think it would be worth / useful to mention somewhere that the benches can be run with `cargo bench` (or the exact command you were using). Also, `state-serialize.md` should be updated as it mentions about `basic.rs` which is now renamed.

Reply:
This turns off the autodiscovery of benchmark files as we are linking them explicitly by main.rs. I would not document this at it is in the criterion docs.

Reply:
Yep, it was probably not the best place to add my comment. The idea was to mention somewhere (in the benchmark doc maybe ?) how to run the benchmarks, some people may not know about `cargo bench`, but I suppose this can be done in a subsequent PR.

Reply:
I am planning to run them on each PR or in the long duration CI, but that needs more work. Basically they can be run as you would run any bench: `cargo bench`. The problem is there is no value in this if we do not have a baseline.